schemas/data/poi-exits.schema.json

```diff
@@ -41,6 +41,9 @@
   "allOf": [
     {
       "$ref": "#/definitions/metric"
```

Comment:
This looks like an error too, it should be "$ref": "#/definitions/metricdefinition"